sigs.yaml

```diff
@@ -0,0 +1,11 @@
+sigs:
+  - dir: sig-template
+    name: SIG Template
+    mission_statement:
+      Sample mission statement
+    charter_link: sig-template/charter.md
```

Comment:
Might also be worth having something like a `chat_link` for the corresponding Slack channel?

Reply:
sounds good
also `label` was taken from the kubernetes commuity yaml. You're correct, from there they refer to a label for issues/PRs